dockerfiles/startup.sh

```diff
@@ -2,4 +2,5 @@
 
 trap /home/centos/streamwebs/dockerfiles/cleanup.sh EXIT
 bower install --allow-root
+yes "yes" | python /home/centos/streamwebs/streamwebs_frontend/manage.py migrate
```

Comment:
NO!

Do not make migrations every time, that is the whole point of this PR - we have a migration that sets the current state of the db - if we change the db, we will do that with a new migration, and add the new migration to the repo. This is very important for deployment, it is the only way to sensibly migrate the existing staging or production database with new changes.